### ISSUE TITLE
Crash on Turkish OS.

### ISSUE THREAD

**Issue by ersinisre** [MagicBees/MagicBees#147]
I used to have crashes on FTB packs like "infinity" and i realized that when i disable magicbees mod it works perfect. After that i found out that it is caused by my system language being Turkish. Adding parameter " -Duser.language=en -Duser.country=US " worked perfect, i have been playing for 2 years without any problem. But now, the server community has created a "community mod pack" it is similar to infinity and it includes magicbees to. The problem is, they are using technic launcher for it. And i cant launch the mod pack. Deleting Magicbees solves the problem. But adding parameter  
" -Duser.language=en -Duser.country=US " does not work.  Please help me with this.
 Here is the crash log.
[crash-2016-07-09_03.49.52-client.txt](https://github.com/MagicBees/MagicBees/files/355504/crash-2016-07-09_03.49.52-client.txt)

**Comment by mezz** [comment]
This probably has to do with Turkish uppercase and lowercase for I. Sometimes mods will "toLowercase()" on English words, which uses the player's language instead of English and causes issues.

**Comment by ersinisre** [comment]
Yes, that's what I have guessed. Is it impossible to solve without parameters?  Because Technic Launcher won't load parameters properly.  I solved this with MultiMC;  but I have to do all that work again      everytime an update comes out.

**Comment by mezz** [comment]
I ran into this issues in Forestry and reported it to many others, but not everything has been fixed. 
Magic Bees development is at a standstill right now, since mods are moving to 1.10 and this project depends on so many of them. I think your best option is to use the parameters, sorry.

**Comment by xJon** [comment]
A bit late but for anyone who comes across this, you can use https://www.curseforge.com/minecraft/mc-mods/localefixer (or follow the recommendations on that page).
